website/docs/getting-started/packages.md

```diff
@@ -0,0 +1,12 @@
+---
+sidebar_position: 4
```

Comment:
I don't know if this is intended to be here 
![image](https://user-images.githubusercontent.com/29845919/209672983-8450fbfc-9e71-4a5c-99de-f65a37e18533.png)

Reply:
Yes. It's the way we define sidebar ordering.